test/api/api_test.go

```diff
@@ -360,13 +360,17 @@ func RunRequests(
 
 	// Optionally check API server stdout/stderr
 	expectedStdoutPath := "expected-server-stdout"
+	expectedStderrPath := "expected-server-stderr"
+	if testDirFs.IsFile(expectedStdoutPath) || testDirFs.IsFile(expectedStderrPath) {
+		// Wait a while the server logs everything for previous requests.
+		time.Sleep(100 * time.Millisecond)
```

Comment:
Spadol nahodne API E2E test: https://github.com/keboola/keboola-as-code/actions/runs/3067518426/jobs/4953893066#step:8:1399

Musime chvilu pockat, kym API server stihne vsetko zalogovat, a az tak to mozeme skontrolovat.